server/apps/prepopulate/app_initialize.py

```diff
@@ -50,19 +54,20 @@ class AppInitializeWithDataCommand(superdesk.Command):
     """
 
     option_list = [
-        superdesk.Option('--entity-name', '-n', dest='entity_name', default='')
+        superdesk.Option('--entity-name', '-n', dest='entity_name', default=''),
+        superdesk.Option('--profile', '-p', dest='profile')
```

Comment:
well in case you don't use docker to deploy, it would be useful to point this to different directory outside of git repo so you don't have to deal with conflicts when updating. with docker it doesn't matter, but there is no need for multiple profiles, you can just copy to root folder